### ISSUE TITLE
Broken links?

### ISSUE THREAD

**Issue by luc-tielen** [patrickt/recschemes#7]
At the bottom of the page, the link should be: https://blog.sumtypeofway.com/posts/recursion-schemes-part-3.html
Instead I get a 404 error, it tries to link me to: http://blog.sumtypeofway.com/recursion-schemes-part-iii-folds-in-context/

**Comment by luc-tielen** [comment]
Also the link from part 3 to 4 is incorrect.

**Comment by patrickt** [comment]
Yeah, these are all kinds of screwed up ever since I ditched my old CMS. Hoping to get to it this weekend.

**Comment by luc-tielen** [comment]
Awesome! 😄 
I also noticed there's no link from 4 to 5. 
(Good thing the URLs are straightforward 😅)
